quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -209,15 +218,20 @@ const PARAMS: Params = Params {
 
     probe_rtt_inflight_target_bdp_fraction: 0.5,
 
+    // The default period for entering PROBE_RTT. 10s
     probe_rtt_period: Duration::from_millis(10000),
 
+    // The default time to spend in PROBE_RTT mode
     probe_rtt_duration: Duration::from_millis(200),
 
     initial_max_ack_height_filter_window: 10,
 
     inflight_hi_headroom: 0.15,
 
-    loss_threshold: 0.015,
+    // Tolerate a loss of 2%
+    //
+    // https://github.com/google/quiche/blob/98c9cdb4cd17ea043243037bfdee3cdf024cab54/quiche/common/quiche_protocol_flags_list.h#L146-L149
+    loss_threshold: 0.02,
```

Comment:
Loss threshold seems too conservative.

Reply:
I don't know what the loss rate for cubic tends to be.  Allowing 2% loss instead of 1.5% seems worth experimenting with, but it doesn't seem that conservative to me.